### DISCUSSION TITLE
[UP SMF] Dynamic Question Paper Creation from Question Bank

### DISCUSSION THREAD

**Discussion by abhay-samagra** [Sunbird-coKreat/Community#90] [Feature Requests]
**Context:**
In UPSMF System, Questions are crowd sourced into question bank. Users with creation access in the system create Blueprints. Blueprints are form of question Set which do not have questions explicitly marked against question set, but rather have a criteria to fetch these questions from question bank.

**Problem Statement:**
For UPSMF (Uttar Pradesh State Medical Faculty), Platform should enable user to automate fetching of questions into question-set based on taxonomy association. 

**Solution:**
In order to enable pulling of questions against a blue print at runtime, there is a need to build an extension to the system to enable the existing system to explicitly automate the fetching of questions into the Question-Set(s).

As part of Mission Niramaya in UP, under the department of Medical Education, Govt of UP, the solution has been implemented by the State Team leveraging the source code of Sunbird ED.

**Request:**
Please suggest the way forward to get this checked back into the source code.

The details of the solution approach are marked in the documentation [here](https://project-sunbird.atlassian.net/wiki/spaces/UPSMF/pages/3261890567/Dynamic+Question-Set)

@manimegalai-smohan , @vinukumar-vs , 
cc: @surendrasinghs

**Comment by kameshbhr** [comment]
This should be part of Sunbird inQuiry, not coKreat. @PoojaJManjooran @sajeshkayyath Is there a plan to get this into inQuiry?

**Reply by abhay-samagra** [reply]
Thanks for clarifying Kamesh.
Found the relevant ticket in [InQuiry](https://github.com/orgs/Sunbird-inQuiry/discussions/50)
Closing it here and linking this in that ticket.
